cardano-config/src/Cardano/Config/Types.hs

```diff
@@ -87,6 +88,7 @@ data NodeCLI = NodeCLI
   , configFp :: !ConfigYamlFilePath
   , validateDB :: !Bool
   , shutdownIPC :: !(Maybe Fd)
+  , shutdownOnSlotSynced :: !MaxSlotNo
```

Comment:
This could also be useful for mock protocols. Is there a common place for shut arguments?

Reply:
Done.